package.json

```diff
@@ -19,6 +19,7 @@
   },
   "packageManager": "pnpm@8.6.0",
   "devDependencies": {
+    "@types/prettier": "^2.7.3",
```

Comment:
Should this be a dependency of the website? Typescript isn't a global dependency, and we only have a typescript config in the website?

Reply:
yeah makes sense. i had it there because i was working on both the site and chatbot, but keeping it separate makes sense